lib/dal/src/management/prototype.rs

```diff
@@ -1032,6 +1049,23 @@ impl ManagementPrototype {
     }
 }
 
+async fn handle_failed_management_func(
+    ctx: &DalContext,
+    manager_component_id: ComponentId,
+    id: ManagementPrototypeId,
+    func_run_id: FuncRunId,
+) -> Result<(), ManagementPrototypeError> {
+    FuncRunner::update_run(ctx, func_run_id, |func_run| {
+        func_run.set_action_result_state(Some(si_events::ActionResultState::Failure));
+    })
+    .await?;
+    WsEvent::management_func_executed(ctx, id, manager_component_id, func_run_id)
```

Comment:
Should we also use one of the `WsEvent::management_operations*` three events too?

Reply:
This is before operations have begun so i think no